lang/java/reef-tests/src/test/java/org/apache/reef/tests/applications/vortex/addone/AddOneTest.java

```diff
@@ -85,7 +85,7 @@ public void testVortexAddOneCallback() {
         .set(VortexMasterConf.WORKER_MEM, 64)
         .set(VortexMasterConf.WORKER_CORES, 4)
         .set(VortexMasterConf.WORKER_CAPACITY, 2000)
-        .set(VortexMasterConf.VORTEX_START, AddOneTestStart.class)
+        .set(VortexMasterConf.VORTEX_START, AddOneCallbackTestStart.class)
```

Comment:
@yunseong Opened 1149 for it, which is done in #789. Please have a look at that as well, thanks!